### ISSUE TITLE
'out.R1.fq.gz' is not a writable file, quit now

### ISSUE THREAD

**Issue by xuzhougeng** [OpenGene/fastp#51]
When I use the  version 0.13.0, and run the follwing command 
```
fastp -z 1 -i test_1.fq -I test_2.fq --out1 out.R1.fq.gz --out2 out.R2.fq.gz
```
and then I met the problem"'out.R1.fq.gz' is not a writable file, quit now"

After `touch out.R1.fq.gz` firstly ,  this problem will be overcome.

**Comment by sfchen** [comment]
That's caused by a bug of C system API, which acts differently on different Linux systems.

I just fixed it, could you please try the latest v0.13.1 ?

BTW, what's your system?

**Comment by xuzhougeng** [comment]
The v0.13.1 can run successfully.
 
AND, This is my linux system information.
LSB Version:	:core-4.1-amd64:core-4.1-noarch:cxx-4.1-amd64:cxx-4.1-noarch:desktop-4.1-amd64:desktop-4.1-noarch:languages-4.1-amd64:languages-4.1-noarch:printing-4.1-amd64:printing-4.1-noarch
Distributor ID:	CentOS
Description:	CentOS Linux release 7.4.1708 (Core) 
Release:	7.4.1708
Codename:	Core
